xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -2244,3 +2246,9 @@ std::shared_ptr<CPVREpgInfoTag> CPlayerOperations::GetCurrentEpg()
 
   return currentChannel->GetEPGNow();
 }
+
+JSONRPC_STATUS CPlayerOperations::GetDatabaseName(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  result = CServiceBroker::GetDatabaseManager().GetFullDatabaseName(CViewDatabase::GetDefaultBaseDBName());
+  return OK;
+}
```

Comment:
Implementation should not be in `CPlayerOperations`, because view database being completely unrelated to player.